index.js

```diff
@@ -1,16 +1,55 @@
 class SortedList {
-  constructor() {}
+  constructor(items,length) {
+    this.items=[];
+    this.length= this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.item=item;
+    this.items.push(item);
+    this.length +=1;
+    this.items.sort((a,b)=>{a-b});
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos>=this.items.length) 
+    {throw new Error('OutOfBounds')}
+    else {return this.items[pos]}
+  }
 
-  max() {}
+  max() {
+    if (this.items=[])
```

Comment:
with just one `=` sign you are assigning a new value to the variable (in this case, you are setting `this.items` to an empty array every time I call the `max` function

Reply:
if you already have a sorted array (because of the add method), you can just return the last element